### DISCUSSION TITLE
无法在CLion中正确运行

### DISCUSSION THREAD

**Discussion by WangShuoCode** [mayerui/sudoku#15] [Q&A]
你好，mayerui，我是一个在校大学生，希望能通过您的项目来学习。
我clone您的代码后，在WSL中编译运行正常，然后我打开CLion来运行，运行成功了，但是Clion使用是内置的shell，无法正确的显示。
![image](https://user-images.githubusercontent.com/67184655/111412926-3ef2da80-8718-11eb-9461-4928c8a27a45.png)
我想知道作者是采用什么工具来编译此项目的，以及如何解决此问题，谢谢。

**Comment by mayerui** [comment]
你好，最近比较忙没有看到这个Discuss。我开发这个项目的工具是VSCode+CMake，运行这个项目则通过VSCode内置终端或者Windows Terminal。我看你编译后二进制文件名后缀是exe，应该是在Windows平台运行的，那么你可以检查一下乱码是否由字符编码引起：sudoku项目里的中文都是utf-8编码格式，而Windows默认编码应该是gbk。
我有后续计划把这个项目中输出的中文都改成英文，应该能免去这种不必要的麻烦，你如果有兴趣的话欢迎提交pr :)
